test/log-generators/install-cronjobs.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Create a CronJob that run every minute, creating an Apache log file between 128 and 512 lines long.
+# The pods for the last 1000 successful runs and 50 failed runs will be kept on the cluster, any logs
+# over those thresholds are automatically cleaned up by kubernetes.
+#
+# The job sleeps at random intervals for 1-3 seconds to get different timestamps in the log.
+
+namespace="generate-logs-cronjobs"
+
+kubectl create ns ${namespace}
+
+kubectl -n ${namespace} apply -f - << EOF
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: generate-log
+spec:
+  schedule: "* * * * *"
+  successfulJobsHistoryLimit: 1000
+  failedJobsHistoryLimit: 50
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: generate
+            image: redhat/ubi8
```

Comment:
When I was investigating how to approach the logging integration part, I saw that there are some images available to create random logs, I think that we could take advantage of one of those:
* https://github.com/chentex/random-logger -- https://hub.docker.com/r/chentex/random-logger
* https://github.com/mingrammer/flog -- https://hub.docker.com/r/mingrammer/flog

The first one has more pulls in Dockerhub but the second one has more stars and forks, but I think that both of them could fit.

Reply:
I looked at both of those, and concluded that for what I need now the script is sufficient.  We could of course update this later if we decide we need the additional log generating functionality that these images might provide.

Reply:
The advantages that I see of using one of those images are:
* Better readability, it's a short script but bash isn't a verbose language so if we stick with the script I would want to add some comments over it to make things clearer
* Get rid of the embedded code in the container command
* Code already being maintained and used by a community
* No hard dependency for us, as it's just an image pulled the same as `redhat/ubi8`
* The change is quickly done even if we want to roll it back in the future --> this could be a point for keeping the script right now also, because as you mention we could update this in the future, but given the previous points I would rather try this out now